### PR TITLE
pysimplegui v4.15.2

### PR DESCRIPTION
**Pull request by Silun**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Silun** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 {% set styledName = "PySimpleGUI" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz"
-  sha256: "c3edb437bd50da95b9d7d205778ec85a595b61cfa9a6aeb2779d3b3da9a725ea"
+  url: "https://files.pythonhosted.org/packages/b2/d0/c20c020af7d2e61ab61ba2b2c9299b5eda75ab5615498733f1a04d8996c1/PySimpleGUI-{{ version }}.tar.gz"
```
Comment (chrisburr) [inline recipe/meta.yaml]: You should change this back to the predictable URL:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz
```

**Comment by Silun** [comment]
Thank you, I accepted your proposed change. Weirdly, the checks are successful this time - I originally changed it to that other URL because they kept failing. Well, it doesn't matter now.

So what happens now? Just wait? There are multiple update requests that didn't go through already, and the maintainer has been inactive since November, it seems.

**Comment by chrisburr** [comment]
Occasionally PyPI has issues resolving the predicable URLs so it might have been that.

As for now, we can ping the maintainers of this feedstock to ask them to merge this. As you're new to conda-forge pinging teams isn't available to you (it's a GitHub limitation), but you can ask the bot to do it for you using:

@conda-forge-admin, please ping team

If they don't respond within a couple of days, ping me and I'll ask someone from the core team[1] to merge this (I don't have rights to do it myself).

On that note, would you like to add yourself as a maintainer of this feedstock? If so, you can add your username to the end of the `meta.yaml` and you'll be automatically given privileges when this is merged.

[1] Again GitHub limitations mean you can't ping them yourself 😞

**Comment by Silun** [comment]
Hey @chrisburr it has been a few days and there was no response so far. As per your suggestion I have added myself as a second maintainer. Please do what needs doing, as you lined out :)

**Comment by chrisburr** [comment]
@conda-forge/core Can someone merge this? There hasn't been a response from the existing maintainers and @Silun has included themselves as a maintainer for future updates.

**Comment by CJ-Wright** [comment]
@conda-forge-admin please rerender

**Comment by CJ-Wright** [comment]
I think this can go in.

**Pull request merged**